js/create.js

```diff
@@ -0,0 +1,71 @@
+import { request } from "../js/HTTP_request_base.js";
+
+const url = "https://api.noroff.dev/api/v1/social/posts";
+
+////////// CREATE POST
+async function createPost(data) {
+  try {
+    const result = await request(
+      url,
+      "POST",
+      data,
+      localStorage.getItem("token")
+    );
+
+    if (!result) {
+      throw new Error("Failed to create post");
```

Comment:
Make sense. But this error will only show up in the browser's console. It would be better to display a notification on the UI.

Reply:
The HTTP_requst_base.js provides intel to the user about what went wrong with an alert message. 
Example: If the user tries to create a post when the JWT is not present, the user is alerted with the following message: 
"Failed to create post: Request failed with status 401: No authorization header was found".

But now I've also added an inline HTML error message as prompted.

Reply:
By default the user should get alerted once an error occur. But I now have added inline error message for the user as well.